pkg/resource/feature_group/testdata/feature_group/cr/fg_cmd_invalid_before_create.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: FeatureGroup
+metadata:
+  name: unit-testing-feature-group
+spec:
+  eventTimeFeatureName: EventTime
+  featureDefinitions:
+  - featureName: TransactionID
+    featureType: Integral
+  - featureName: EventTime
+    featureType: Fractional
+  featureGroupName: !unit-testing-feature-group
```

Comment:
nit: add a comment above this. invalid name or change the name to say so

Reply:
This name was changed in PRs #66 and #76. It has now been added to this one as well.